node_modules/gh-series/lib/authz.js

```diff
@@ -38,68 +38,78 @@ var canCreateSeries = module.exports.canCreateSeries = function(ctx, callback) {
 };
 
 /**
- * Determine whether or not the user in context can update a serie
+ * Determine whether or not the user in context can manage a series
  *
  * @param  {Context}    ctx                         Standard context object containing the current user and the current application
- * @param  {Serie}      serie                       The serie that the current user wants to update
+ * @param  {Serie}      series                      The series that the current needs to manage
  * @param  {Function}   callback                    Standard callback function
  * @param  {Object}     callback.err                An error that occurred, if any
  * @param  {Boolean}    callback.canUpdate          `true` if the user in context has the appropriate permission. `false` otherwise
  */
-var canUpdateSerie = module.exports.canUpdateSerie = function(ctx, serie, callback) {
-    // Anonymous users cannot update a serie
+var canManageSeries = module.exports.canManageSeries = function(ctx, series, callback) {
+    // Anonymous users can never manage a series
     if (!ctx.user) {
         return callback(null, false);
 
-    // App administrators (or global admins) can update a serie
-    } else if (ctx.user.canAdmin(serie.AppId)) {
+    // Application administrators (or global admins) can always manage a series
+    } else if (ctx.user.canAdmin(series.AppId)) {
         return callback(null, true);
     }
 
-    // At this point we need to check if the user is a member of the serie's group
-    GroupsDAO.isMember(serie.Group, ctx.user, callback);
+    // At this point we need to check if the user is a member of the series' group
+    GroupsDAO.isMember(series.Group, ctx.user, callback);
+};
+
+/**
+ * Determine whether or not the user in context can update a series
+ *
+ * @param  {Context}    ctx                         Standard context object containing the current user and the current application
+ * @param  {Serie}      series                      The series that the current user wants to update
+ * @param  {Function}   callback                    Standard callback function
+ * @param  {Object}     callback.err                An error that occurred, if any
+ * @param  {Boolean}    callback.canUpdate          `true` if the user in context has the appropriate permission. `false` otherwise
+ */
+var canUpdateSerie = module.exports.canUpdateSerie = function(ctx, series, callback) {
```

Comment:
Yes, but I'm consciously trying not to let the refactor spin out of control. I'm only changing `serie`->`series` in lines I'm touching or aren't triggering further refactor.

I'll make a pass through once some of the bigger PRs have gone through (time-permitting).